bool physical
bool simulated
---
bool status
string message
